subsystems/intakesubsystem.py

```diff
@@ -68,22 +70,73 @@ def __init__(self) -> None:
             .publish()
         )
 
+        self.intakeL1SpeedPublisher = (
+            NetworkTableInstance.getDefault()
+            .getFloatTopic(constants.kIntakeL1SpeedKey)
+            .publish()
+        )
+        self.intakeL1SpeedPublisher.set(constants.kIntakeL1MotorSpeed)
+
+        self.intakeL1SpeedGetter = (
+            NetworkTableInstance.getDefault()
+            .getFloatTopic(constants.kIntakeL1SpeedKey)
+            .subscribe(constants.kIntakeL1MotorSpeed)
+        )
+
+        self.intakeL2ThroughL4Publisher = (
+            NetworkTableInstance.getDefault()
+            .getFloatTopic(constants.kIntakeL2ThroughL4SpeedKey)
+            .publish()
+        )
+        self.intakeL2ThroughL4Publisher.set(constants.kIntakeL2ThroughL4MotorSpeed)
+
+        self.intakeL2ThroughL4SpeedGetter = (
+            NetworkTableInstance.getDefault()
+            .getFloatTopic(constants.kIntakeL2ThroughL4SpeedKey)
+            .subscribe(constants.kIntakeL2ThroughL4MotorSpeed)
+        )
+
+        self.intakeCoralSpeedPublisher = (
+            NetworkTableInstance.getDefault()
+            .getFloatTopic(constants.kIntakeCoralKey)
+            .publish()
+        )
+        self.intakeCoralSpeedPublisher.set(constants.kIntakeMotorSpeed)
+
+        self.intakeCoralSpeedGetter = (
+            NetworkTableInstance.getDefault()
+            .getFloatTopic(constants.kIntakeCoralKey)
+            .subscribe(constants.kIntakeMotorSpeed)
+        )
+
+        self.elevator = ElevatorSubsystem()
```

Comment:
this creates a new instance of the elevator subsystem rather than referring to the initial one, get the elevator state from networktables